kivy/uix/gridlayout.py

```diff
@@ -307,8 +308,16 @@ def on_children(self, instance, value):
                 'Too many children in GridLayout. Increase rows/cols!')
 
     @property
-    def _fills_row_first(self):
-        return self.orientation[0] in 'lr'
+    def fills_row_first(self):
+        return self.orientation[0] not in 'tb'
+
+    @property
+    def fills_from_left_to_right(self):
```

Comment:
Perhaps these should be private properties (with leading underscore). And the same for the other properties.

Reply:
> Perhaps these should be private properties

got it.

> we default it to e.g. cols being one.

Yes, that's the behavior before #6741, and thus probably is the safest option I believe.